.env.test.optimismSepolia

```diff
@@ -0,0 +1,58 @@
+# An optional (but highly recommended) RPC URL for the L1 Chain
+VITE_L1_PUBLIC_RPC_URL=https://rpc.sepolia.org
+# The address of the token on the parent chain that will be used as the gas token on the rollup - does not need to be on an "L1"
+# Alternatively, set to 0x0000000000000000000000000000000000000000 to use the bridge for native currency
+VITE_L1_CUSTOM_GAS_TOKEN_ADDRESS= 0x0000000000000000000000000000000000000000
+# The address of the "L1" (parent) contract that will be used to deposit/withdraw funds from the rollup
+VITE_L1_OPTIMISM_PORTAL_ADDRESS=0x16Fc5058F25648194471939df75CF27A2fdC48BC
+# The name of the gas token
+VITE_L1_CUSTOM_GAS_TOKEN_NAME=Native Token
+# The symbol of the gas token
+VITE_L1_CUSTOM_GAS_TOKEN_SYMBOL=ETH
+# The number of decimals the gas token has - this _must_ be 18 per OP Stack requirements
+VITE_L1_CUSTOM_GAS_TOKEN_DECIMALS=18
+# chain ID of the "L1" (parent) chain
+VITE_L1_CHAIN_ID=11155111
+# URL of the official branding icon for the "L1" (parent) chain
+VITE_L1_ICON_URL=https://avatars.githubusercontent.com/u/6250754?s=200&v=4
+# Background color for the icon of the "L1" (parent) chain
+VITE_L1_ICON_BACKGROUND_COLOR="#000"
+
+# chain ID of the "L2" (rollup) chain
+VITE_L2_CHAIN_ID=11155420
+# name of the "L2" (rollup) chain
+VITE_L2_CHAIN_NAME=OP Sepolia 
+# separate multiple values with commas; no spaces. single value with no commas is fine.
+VITE_L2_RPC_URLS=https://sepolia.optimism.io
+# URL of the block explorer for the "L2" (rollup) chain
+VITE_L2_EXPLORER_URL=https://optimism-sepolia.blockscout.com
+# The address of the output oracle contract on the "L2" (rollup) chain
+VITE_L2_OUTPUT_ORACLE_ADDRESS=0x90E9c4f8a994a250F6aEfd61CAFb4F2e895D458F
+# The address of the 'disputeGameFactory' contract on the "L2" (rollup) 
+## ++ for Fault proofs - cc.https://docs.optimism.io/stack/fault-proofs/explainer
+VITE_DISPUTE_GAME_FACTORY_ADDRESS=0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1
```

Comment:
Can we add a placeholder for this to `.env.example`? That's the canonical reference for what environment variables should be included.

Reply:
Done ✅